File: spec/php/NavParentFalse2Test.php
<?php
namespace Kaitai\Struct\Tests;

class NavParentFalse2Test extends TestCase {
    public function testNavParentFalse2() {
        $r = NavParentFalse2::fromFile(self::SRC_DIR_PATH . "/fixed_struct.bin");

        $this->assertEquals(80, $r->parentless->foo);
    }
}

File: spec/go/nav_parent_false2_test.go
package spec

import (
	"runtime/debug"
	"os"
	"testing"

	"github.com/kaitai-io/kaitai_struct_go_runtime/kaitai"
	"github.com/stretchr/testify/assert"

	. "test_formats"
)

func TestNavParentFalse2(t *testing.T) {
	defer func() {
		if r := recover(); r != nil {
			debug.PrintStack()
			t.Fatal("unexpected panic:", r)
		}
	}()
	f, err := os.Open("../../src/fixed_struct.bin")
	if err != nil {
		t.Fatal(err)
	}
	s := kaitai.NewStream(f)

	var r NavParentFalse2
	err = r.Read(s, &r, &r)
	if err != nil {
		t.Fatal(err)
	}

	assert.EqualValues(t, 80, r.Parentless.Foo)
}
